src/config/api.rs

```diff
@@ -64,6 +64,8 @@ pub struct ApiConfig {
     pub default_model: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub timeout: Option<u16>,
```

Comment:
```suggestion
   /// client timeout in seconds
    pub timeout: Option<u16>,
```

Was unsure about renaming that to `timeout_seconds` or adding a comment.

Reply:
Good point, I'd vote for timeout_seconds myself since then the meaning is clear and unambiguous on the config file itself

Reply:
Sounds great to me!